src/private/teams.ts

```diff
@@ -0,0 +1,49 @@
+import { sendMessageToParent } from '../internal/communication';
+import { ensureInitialized } from '../internal/internalAPIs';
+import { FrameContexts, SdkError } from '../public';
+
+/**
+ * Namespace to interact with the files specific part of the SDK.
+ *
+ * @private
+ * Hide from docs
+ */
+export namespace teams {
+  export enum ChannelType {
+    Regular = 0,
+    Private = 1,
+    Shared = 2,
+  }
+
+  export interface TeamsChannelInfo {
+    siteUrl: string;
+    objectId: string;
+    folderRelativeUrl: string;
+    displayName: string;
+    channelType: ChannelType;
```

Comment:
Do you need all of these properties?

Reply:
Yes, these are all the properties we currently need for our feature :)